src/offliner.js

```diff
@@ -228,6 +228,9 @@
       case 'xpromise':
         this._receiveCrossPromise(msg.id, msg.order);
         break;
+      case 'checkForActivationPending':
```

Comment:
> Nit: I think the correct wording (here and elsewhere) is check for pending activation, but I'm not totally sure :smile:

They're both reasonably correct! "Check for activation pending" can mean "check for an activation that is pending," while "check for pending activation" can mean "check for a pending activation." And those are the same thing. In both cases, the word "pending" is being used as an adjective, to modify the noun "activation."

However, "check for activation pending" could be interpreted to mean "check that the thing that is pending is an activation," which would be incorrect here. And the most important part of the sentence is "pending," as it's key to the check: whether or not there is an activation, the key is to check whether or not the activation (if any) is pending.

So I would probably go with "checkForPendingActivation".